### DISCUSSION TITLE
is it possible to implement this library using transform 2d ?

### DISCUSSION THREAD

**Discussion by sunchayn** [idiotWu/smooth-scrollbar#345] [Q&A] [answered]
Currently this library using `transform3d` to do the magic thought for my use case the z value is always 0 this behavior is breaking my z-indexes as it's a known issue when using `transform3d`. 
I'm wondering why not using `transform(x,y)` instead it achieves the same thing, is there any technical needs for that ?

**Comment by oguilleux** [comment, marked as answer]
Hi @mazentouati,

This article explains a lot of things about performance and using translateZ or transform3D is one of the key factors : https://www.smashingmagazine.com/2016/12/gpu-animation-doing-it-right/
